cfgov/unprocessed/css/apps/owning-a-home/brand-footer.less

```diff
@@ -0,0 +1,62 @@
+.brand-footer {
+    background-color: @gray-5;
+    padding-top: @grid_gutter-width;
+    padding-bottom: 60px;
+    border-top: 1px solid @gray-40;
+
+    .title-link {
+        border-bottom-width: 1px;
+    }
+
+    &_img {
+        text-align: center;
+
+        .respond-to-min( @bp-sm-min, {
+            text-align: left;
+        } );
+
+        a:after {
+            display: none !important;
```

Comment:
Avoid `!important` if possible.

Reply:
This code was no longer needed, so I removed it.